src/domain-services/location/graphql/types.ts

```diff
@@ -25,7 +25,7 @@ export default class Location extends BaseType {
   @MaxLength(255)
   name?: string;
 
-  @Field(() => Int)
+  @Field(() => Number)
```

Comment:
Why this changed to `Number`?

Reply:
This is in order to not  depend (too much) on the types provided from 'type-graphql' since we are not sure if we want to keep it in a long term.